extensions/applicationinsights-debugplugin-js/src/components/LogEntry.ts

```diff
@@ -81,7 +81,7 @@ export class LogEntry {
                     ms = "0" + ms;
                 }
 
-                ms = ms.replace(/(.)(\d{3})+$/g, '$1.$2s')
+                ms = ms.replace(/(.)(\d{3}){1}$/g, '$1.$2s')
```

Comment:
The regex changes are security related and a little more complicated and the changes are 2 fold
- Unnecessary escaping (additional \ when not needed), this *could* allow someone to "fiddle" with the resulting regex (in memory) to change it's behavior
- When using * and +, depending on the input string unconstrained matching can cause the regex to use excessive memory, cpu or just fail.